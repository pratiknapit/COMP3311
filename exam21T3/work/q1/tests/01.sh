psql property -c 'select * from q1;'
