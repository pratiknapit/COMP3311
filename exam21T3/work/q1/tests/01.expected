    date    |  price  |   type    
------------+---------+-----------
 2021-11-09 | 1816000 | Townhouse
 2021-11-09 | 1859000 | Townhouse
 2021-11-09 | 2154000 | House
 2021-11-09 | 2230000 | House
 2021-11-09 | 2665000 | House
 2021-11-09 | 2833000 | House
 2021-11-09 | 2836000 | House
(7 rows)
